distiller/quantization/quantizer.py

```diff
@@ -176,6 +176,11 @@ def prepare_model(self):
         if self.optimizer:
             optimizer_type = type(self.optimizer)
             new_optimizer = optimizer_type(self._get_updated_optimizer_params_groups(), **self.optimizer.defaults)
+            # we need explicitly add the initial_lr and lr attribute
+            for group, new_group in zip(self.optimizer.param_groups, new_optimizer.param_groups):
+                if 'initial_lr' in group.keys():
+                    new_group['initial_lr'] = group['initial_lr']
+                new_group['lr]=group['lr]
```

Comment:
Missing closing quotes
```suggestion
                new_group['lr'] = group['lr']
```